cookbooks/aws-parallelcluster-platform/resources/spack/spack_ubuntu.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
```

Comment:
This shouldn't be just plain ubuntu.  We have deprecated ubuntu 18 so this is really just ubuntu 20